ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -508,3 +508,27 @@ def test_tensorflow_l2_normalize(
         x=x[0],
         axis=axis,
     )
+
+
+# sqrtm
+@handle_cmd_line_args
+@given(
+    dtype_and_input=_get_dtype_and_matrix(),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.linalg.sqrtm"
+    ),
```

Comment:
Hi there!

Thanks for the contribution.

First thing: yout need to assign strategies for all inputs to `test_tensorflow_sqrtm` in the @given decorator.

Reply:
Not sure I understand exactly what you mean, could you explain further? Should I check for all valid datatypes and inputs?

Reply:
You need to define each of the arguments for the test function in the given decorate, define the hypothesis strategy. Please have a read of of the hypothesis test writing [section](https://lets-unify.ai/ivy/deep_dive/ivy_tests.html) of the docs for more details :)

Reply:
> You need to define each of the arguments for the test function in the given decorate, define the hypothesis strategy. Please have a read of of the hypothesis test writing [section](https://lets-unify.ai/ivy/deep_dive/ivy_tests.html) of the docs for more details :)

My apologies, I was incorrect about this. These arguments are handled by the `@handle_cmd_line_args` decorator.